### PR TITLE
feat: Improve Korean data-i18n

### PR DESCRIPTION
**Pull request by tincansimagine**
## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

### CODE REVIEW

File: public/locales/ko-kr.json [hunk 1]
```diff
@@ -1039,27 +1057,27 @@
     "Move message up": "메시지를 위로 이동",
     "Move message down": "메시지를 아래로 이동",
     "Enlarge": "확대",
-    "Welcome to SillyTavern!": "SillyTavern에 오신 것을 환영합니다!",
-    "welcome_message_part_1": "읽기",
+    "Welcome to SillyTavern!": "SillyTevern에 오신 것을 환영합니다!",
```
Comment (Cohee1207) [inline public/locales/ko-kr.json]: Typo

File: public/locales/ko-kr.json [hunk 2]
```diff
@@ -1277,24 +1293,25 @@
...
-    "Source": "원천",
+    "Source": "근원",
+    "Horde": "Horde",
     "sd_auto_url": "예: {{auto_url}}",
     "Authentication (optional)": "인증(선택사항)",
     "Example: username:password": "예: 사용자 이름:비밀번호",
     "Important:": "중요한:",
     "sd_auto_auth_warning_1": "SD 웹 UI를 실행하세요",
-    "sd_auto_auth_warning_2": "깃발! SillyTavern 호스트 시스템에서 서버에 액세스할 수 있어야 합니다.",
+    "sd_auto_auth_warning_2": "Flag! SillyTevern 호스트 시스템에서 서버에 액세스할 수 있어야 합니다.",
```
Comment (Cohee1207) [inline public/locales/ko-kr.json]: At this point I'm wondering if it's intentional

Comment (tincansimagine) [reply public/locales/ko-kr.json]: Ah, I'm so sorry. you wanted to keep the SillyTavern branding, so I did a bulk replacement of the Korean "실리태번" back to the English, and there was a glitch in the process. I'm really sorry for the trouble. I truly didn't mean to cause you any inconvenience.

Comment (tincansimagine) [reply public/locales/ko-kr.json]: Aside from the name error with SillyTavern, there shouldn't be any more mistakes. I've checked it about three times.  I'm so sorry to bother you again.

Comment (Cohee1207) [reply public/locales/ko-kr.json]: Well, fix it so I could merge it.

Comment (tincansimagine) [reply public/locales/ko-kr.json]: I fixed it.

**Review by Cohee1207 — APPROVED** [review]
Looks good now. Thank you.

**Pull request merged**